src/main/java/org/sunbird/cache/service/RedisCacheServiceImpl.java

```diff
@@ -23,7 +23,7 @@ public class RedisCacheServiceImpl implements RedisCacheService {
 	@Override
 	public SBApiResponse deleteCache() throws Exception {
 		SBApiResponse response = new SBApiResponse(Constants.API_REDIS_DELETE);
-		boolean res = redisCache.deleteAllCBExtKey();
+		boolean res = redisCache.deleteAllKey();
```

Comment:
Just keep the old name - do not create confusion -- deleteAllKey sounds dangerous.